test/utils/TestHexUtils.ts

```diff
@@ -198,4 +198,48 @@ describe('HexUtils', () => {
         .toBeRevertedWithoutReason()
     })
   })
+
+  describe('addressToHex()', async () => {
+    for (const addr of [
+      zeroAddress,
+      '0x5ceE339e13375638553bdF5a6e36BA80fB9f6a4F',
+      '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045',
+    ] as const) {
+      it(addr, async () => {
+        const { hexUtils } = await loadFixture(fixture)
+        await expect(hexUtils.read.addressToHex([addr])).resolves.toStrictEqual(
+          addr.slice(2).toLowerCase(),
+        )
+      })
+    }
+  })
+
+  describe('bytesToHex()', async () => {
+    for (let n = 0; n <= 33; n++) {
+      const data = toHex(Uint8Array.from({ length: n }, (_, i) => n + i))
+      it(data, async () => {
+        const { hexUtils } = await loadFixture(fixture)
+        await expect(hexUtils.read.bytesToHex([data])).resolves.toStrictEqual(
+          data.slice(2),
+        )
+      })
+    }
+  })
+
+  describe('unpaddedUintToHex()', async () => {
+    for (const i of [0n, 1n, (1n << 256n) - 1n]) {
```

Comment:
A few other even- and odd-length tests would be good here, in case there's a fencepost error that doesn't affect `0` and `1`.